test_reminder.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date, datetime
+
+# import pytest
+from freezegun import freeze_time
+
+from reminder import Reminder
+
+
+@freeze_time("2023-06-15")
+def test_get_monday():
+    test_date = date(2023, 6, 15)
+    assert Reminder().get_monday(test_date) == date(2023, 6, 12)
+
+
+@freeze_time("2023-06-12")
+def test_is_spring_planning():
+    assert not Reminder().is_sprint_planning()
+
+
+@freeze_time("2023-06-12")
+def test_next_planning():
+    assert Reminder().next_occurance("Sprint planning") == datetime(2023, 6, 19, 15, 30)
+
+
+@freeze_time("2023-06-12")
+def test_next_review():
+    assert Reminder().next_occurance("Sprint review") == datetime(2023, 6, 15, 14, 45)
+
+
+@freeze_time("2023-06-12")
+def test_next_retrospective():
+    assert Reminder().next_occurance("Sprint restrospective") == datetime(
+        2023, 6, 16, 9, 30
+    )
+
+
+def test_next_daily():
```

Comment:
I meant to remove the `time` part of it not the `date` :)

Reply:
I thought it would be necessary but it's not. I added the decorator to this function